cypress/integration/problem_detailed_page.js

```diff
@@ -19,4 +19,10 @@ describe("Problem Detailed Page", () => {
     cy.get(`a[href="/problems/1"]`).first().click();
     cy.contains("Pobierz treść");
   });
+
+  it("shows problem detailed page after clicking on search result tile on mobile and tablet", () => {
```

Comment:
I am just wondering if this test should be here or in the new file for problems page? What do you think?

Reply:
There.

Reply:
Right! 🐈